kafka/Dockerfile

```diff
@@ -9,30 +12,26 @@ RUN microdnf update && \
 
 ENV JAVA_HOME=/usr/lib/jvm/jre-11
 
-RUN groupadd -r stackable --gid=1000; \
-    useradd -r -g stackable --uid=1000 stackable;
-
-ADD stackable /stackable
-RUN chown -R stackable:stackable /stackable
+COPY stackable /stackable
+RUN groupadd -r stackable --gid=1000 && \
+    useradd -r -g stackable --uid=1000 stackable && \
+    chown -R stackable:stackable /stackable
 
 USER stackable
 
 ARG PRODUCT
-ARG SCALA
+ARG SCALA=2.13
```

Comment:
I recommend to not hard code the scala version here but set it in conf.py.

Reply:
@razvan This was just a default that would have been overridden by the matrix build anyway, as far as I can tell. Is there an actual reason to drop Scala 2.12 support here?

Reply:
Not saying I'm particularly pro-2.12, just that I'd like for that to be an explicit decision taken somewhere rather than just a consequence of misunderstandings.. :P

Reply:
yeah it would have been overridden by the build. I thought the default makes it easier to build manually. Im good with both (keeping / leaving it out)